python/ray/serve/_private/deployment_state.py

```diff
@@ -2159,12 +2159,10 @@ def deploy(self, deployment_info: DeploymentInfo) -> bool:
     def autoscale(self, decision_num_replicas: int) -> bool:
         """
         Apply the given scaling decision by updating the target replica count.
-
         Skips if deleting, if `decision_num_replicas` is None, or matches the
         current target. Otherwise updates the state and logs an up/down scaling.
-
```

Comment:
nit: python docstring typically separates the summary from args with an empty line